app/js/app/app.js

```diff
@@ -682,6 +684,119 @@ define([
             $('.joyride-modal-bg').trigger('click');
         });
 
+
+
+        // USER NOTIFICATIONS VIA WEBSOCKETS
+
+        //$rootScope.notificationList = [];
+        //$rootScope.notificationPopups = [];
+        //$rootScope.notificationListLength = 0;
+        $rootScope.currentActivityStreakLength = 0;
+        $rootScope.notificationWebSocket = null;
+        //var signOnTime = Number(new Date());
+        var socketOpen = false;
+
+        $rootScope.openNotificationSocket = function() {
+
+            $rootScope.user.$promise.then(function() {
+             // we are logged in
+
+                // set up websocket and connect to notification endpoint
+                var websocketURI = "ws://localhost:8080/isaac-api/user-alerts/";
```

Comment:
There is a constant somewhere which you will need to use here so that "localhost:8080" gets replaced with the correct domain for test, staging, master and live.